.eslintrc.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
     '@typescript-eslint',
   ],
   rules: {
+    'max-len': [2, { code: 200, tabWidth: 4, ignoreUrls: true }],
```

Comment:
What code does this rule affect?

Reply:
It change the maximum length of each line from 100->200

Reply:
eslint auto remove my braces to keep the promise chain in a single line, but complain my line is too long

Reply:
Cool, but will `tabWidth` change tab to 4 space? Because I kind of like tab to be 2 spaces. Thanks!